uber/automated_emails.py

```diff
@@ -130,6 +130,14 @@ def days_before(days, dt, until=None):
 AutomatedEmail(Group, 'Last chance to pre-assign {EVENT_NAME} group badges', 'reg_workflow/group_preassign_reminder.txt',
          lambda g: state.AFTER_GROUP_PREREG_TAKEDOWN and g.unregistered_badges and (not g.is_dealer or g.status == APPROVED))
 
+# Reminder emails for groups with a remaining balance to pay. These are sometimes necessary for when groups
+# add badges after registering.
+
```

Comment:
Gotcha.  That's fine, we can fix that after M13, since we need to do a bunch of refactoring anyway.